### ISSUE TITLE
Overfitting with encoding layer

### ISSUE THREAD

**Issue by Ethiral** [rgmyr/keras-texture#1]
Hello,
I'm trying to use the encoding layer from DeepTEN example.Even with the simple network architecture my model(network+encoding layer) starts overfitting. When I run my model without encoding it doesn't overfit then.Any help is appreciated.
Thanks!!

**Comment by rgmyr** [comment]
Sorry for the late reply, I didn't notice this issue until just now. Overfitting is typically a network capacity vs. dataset size & quality issue. You can try using more dropout, regularization, or early stopping. Beyond that, it's difficult to say exactly what you should do. It may also just be a simple problem that is best modeled with a less complex network.

**Comment by rgmyr** [comment]
For the `deepten` network specifically, you can also reduce the network capacity by decreasing the value of the `encode_K` and `conv1x1` parameters (which are 32 and 128 by default).
